File: app/src/main/java/com/example/pregaverso/ServizioSottofondo.java
package com.example.pregaverso;

import android.app.Service;
import android.content.Intent;
import android.media.MediaPlayer;
import android.os.IBinder;

import androidx.annotation.Nullable;

public class ServizioSottofondo extends Service {

    MediaPlayer mediap;

    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }

    @Override
    public int onStartCommand(Intent intent, int flags, int startId){
        mediap = MediaPlayer.create(this, R.raw.sottofondostoria);
        if(!mediap.isPlaying()){
            mediap.setLooping(true);
            mediap.start();
        }
        return super.onStartCommand(intent,flags,startId);
    }

    @Override
    public void onDestroy(){
        mediap.stop();
        super.onDestroy();
    }


}

File: app/src/main/java/com/example/pregaverso/servizioSottofondo.kt
package com.example.pregaverso

import android.app.Service
import android.content.Intent
import android.media.MediaPlayer
import android.os.IBinder
/*

class ServizioSottofondo : Service() {
    private var mediap : MediaPlayer = MediaPlayer.create(this, R.raw.sottofondostoria)

    override fun onBind(intent: Intent?): IBinder? {
        return null
    }

    override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
        mediap.isLooping = true
        mediap.start()
        return super.onStartCommand(intent, flags, startId)
    }

    override fun stopService(name: Intent?): Boolean {
        mediap.stop()
        return super.stopService(name)
    }

    override fun onDestroy() {
        mediap.stop()
        super.onDestroy()
    }
}
 */


